pkg/ccl/backupccl/restore_span_covering_test.go

```diff
@@ -42,7 +43,7 @@ func MockBackupChain(length, spans, baseFiles int, r *rand.Rand) []backuppb.Back
 		if i > 0 {
 			backups[i].StartTime = backups[i-1].EndTime
 			if i%3 == 0 {
```

Comment:
I'm confused by a few parts of MockBackupChain: 
 - The `makeTableSpan(uint32(100 + j + (i / 3)))` call above implies that each backup[i].Span[j] is unique, as each gets created with a unique TableID (note that i is included in the TableID calc). Is that intended? I feel like the same spans should get covered over a series of backups.
 - In the line below, I randomized which span gets reintroduced. `TestRestoreEntryCover` continues to pass with this diff; however `BenchmarkCoverageChecks` now fails in the `checkCoverage` call. I don't understand why.

Reply:
hrm. removing the `i` from the input call, seems to have solved the problem 🤷

Reply:
is this concerning? I haven't looked too deeply but does it warrant understanding why the table ID generator affects the test failing/passing?

Reply:
> I feel like the same spans should get covered over a series of backups.

It does with the i/3 doesn't it: at some point, some span, say, 106, starts getting included, and it keeps getting included for at least 3 backups, before i/3 increases (integer division). In fact, I think it stays covered for much more than 3, since the one at the bottom of j slides out and it slides down within j.